polkadot/runtime/rococo/src/genesis_config_presets.rs

```diff
@@ -543,3 +522,13 @@ pub fn get_preset(id: &sp_genesis_builder::PresetId) -> Option<alloc::vec::Vec<u
 			.into_bytes(),
 	)
 }
+
+/// List of supported presets.
+pub fn preset_names() -> Vec<PresetId> {
+	vec![
+		PresetId::from("local_testnet"),
```

Comment:
these name strings are somewhat standardized, right? as in we expect all runtimes to expose exactly `local_testnet`?

Reply:
> these name strings are somewhat standardized, right? as in we expect all runtimes to expose exactly `local_testnet`?

@kianenigma I wouldn't say it is standardized. I think @michalkucharczyk reused an older `chainId` or something [here](https://github.com/paritytech/polkadot-sdk/pull/1256/files#diff-9bad7863337c64e2c77532573edf3352392eaf1b14b28adadaf4cf75125640b5L51-R53), where `local_testnet` had been for a long time. 

Actually, every runtime can define its own relevant values, and we can change it here to whatever we want, possibly standardizing it in a way similar to how `sp_genesis_builder::DEV_RUNTIME_PRESET = "development"` is used.

This `PresetId` value just needs to match [here](https://github.com/paritytech/polkadot-sdk/pull/5327/files#diff-4a9e2a13a5980c8f45146c1f35d31e510d10ca51ef3591dd0a54cc932b7fb1dfR226) to select the preset for genesis:
```
.with_genesis_config_preset_name("local_testnet")
```

Reply:
It seems to me that `local_testnet` meets all the criteria to be treated similar to `DEV_RUNTIME_PRESET` and be standardized.

Both are for development, one is meant to work with single node, the other multi-node.

Is that correct?

Reply:
I would say `development` is for all runtimes - as it this preset is used for benchmarking.

`local_testnet` is rather standard for parity runtimes required for generating some genesis that is widely used in our testing.

We could add it to genesis_builder create, not sure however if it really belongs there.
Maybe we should define const on the _level_ of our _system-parachain_ runtimes.

Reply:
@kianenigma @michalkucharczyk I refactored also "local" / "local_testnet" as a default/standard preset string, because they are used for every runtime,
please check: https://github.com/paritytech/polkadot-sdk/pull/5327/commits/beded8e3f1e0a70a7f35f9772f3759ae7859b2cf